doc/nrf/config_and_build/dfu/nrf70_fw_patch_update.rst

```diff
@@ -0,0 +1,193 @@
+   This partition will be used to store the new nRF70 Series firmware patch during the DFU procedure.
+
+   For example:
+
+    .. code-block:: console
+
+        mcuboot_secondary_2:
+            address: 0x150000
+            size: 0x21000
+            device: MX25R64
+            region: external_flash
+
+#. Update the ``external_flash`` partition to allocate all available memory space to it.
+
+   For example:
+
+    .. code-block:: console
+
+        external_flash:
+            address: 0x171000
+            size: 0x68F000
+            device: MX25R64
+            region: external_flash
+
+.. note::
+    Please note that the actual configuration syntax for the Partition Manager will depend on the specific system and tools being used.
+    The example provided is for illustrative purposes and may need to be adjusted to fit the actual configuration file format and syntax required by the Partition Manager in use.
+
+Configuring build system
+------------------------
+
+To enable the DFU procedure for the nRF70 Series firmware patch, complete the following steps depending on the platform:
+
+* For the nRF5340 DK without the network core:
+
+    1. Set the :kconfig:option:`CONFIG_NRF_WIFI_FW_PATCH_DFU`` Kconfig option to ``y``.
+    #. Set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``2``.
+    #. For the MCUBoot child image, set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``2``.
+
+* For the nRF5340 DK with the network core:
+
+    1. Set the :kconfig:option:`CONFIG_NRF_WIFI_FW_PATCH_DFU`` Kconfig option to ``y``.
+    #. Set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``3``.
+    #. For the MCUBoot child image, set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``3``.
+
+For example, to build the sample with the DFU procedure for the nRF70 Series firmware patch on the nRF5340 DK platform, which includes the network core image, run the following commands:
+.. tabs::
+
+   .. group-tab:: West
+
+      .. code-block:: console
+
+        west build -d nrf5340dk_nrf5340_cpuapp -d -- -DSHIELD=nrf7002ek -DCONFIG_PARTITION_MANAGER_ENABLED=y -DCONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE=y -DCONFIG_NRF_WIFI_FW_PATCH_DFU=y -DCONFIG_UPDATEABLE_IMAGE_NUMBER=3 -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3
+
+   .. group-tab:: CMake
+
+      .. code-block:: console
+
+        cmake -GNinja -Bbuild -DBOARD=nrf5340dk_nrf5340_cpuapp -DSHIELD=nrf7002ek -DCONFIG_PARTITION_MANAGER_ENABLED=y -DCONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE=y -DCONFIG_NRF_WIFI_FW_PATCH_DFU=y -DCONFIG_UPDATEABLE_IMAGE_NUMBER=3 -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3 sample
+        ninja -C build
+
+
+If you want to use the :ref:`ug_multi_image` feature you need to set the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE_MAX_IMAGE_COUNT` kconfig option to ``2`` for the nrf5340 DK platform without network core or to ``3`` for the nrf5340 DK platform with network core.
```

Comment:
```suggestion
If you want to use the :ref:`ug_multi_image` feature, you need to set the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE_MAX_IMAGE_COUNT` Kconfig option to one of the following values:

* For the nRF5340 DK without the network core: ``2``
* For the nRF5340 DK with the network core: ``3``
```
Please double check capitalization of kconfig (> Kconfig) and nrf (> nRF) in plain text parts in the document, as I see some discrepancies.

Reply:
I hope I changed all of them.